web/sites/default/config/webform.webform.report_tnc_crash.yml

```diff
@@ -0,0 +1,458 @@
+title: 'Report a TNC Crash'
+description: ''
+categories: {  }
+elements: |-
+  page_initial_details:
+    '#type': webform_wizard_page
+    '#title': 'Initial Details'
+    report_type:
+      '#type': radios
+      '#title': 'At the time of the crash, the driver was:'
+      '#options':
+        'Operating a tow truck': 'Operating a tow truck'
+        'Operating a taxi, transportation network company (TNC), or other for-hire vehicle': 'Operating a taxi, transportation network company (TNC), or other for-hire vehicle'
+      '#required': true
+    container_tow_company:
+      '#type': container
+      '#states':
+        visible:
+          ':input[name="report_type"]':
+            value: 'Operating a tow truck'
+      report_towing_company:
+        '#type': textfield
+        '#title': 'Which Towing Company?'
+        '#required': true
+      report_towing_vehicle:
+        '#type': radios
+        '#title': 'Were you towing a vehicle at the time of the crash?'
+        '#options': yes_no
+        '#required': true
+    container_pfht_company:
+      '#type': container
+      '#states':
+        visible:
+          ':input[name="report_type"]':
+            value: 'Operating a taxi, transportation network company (TNC), or other for-hire vehicle'
+      report_pfht_company:
+        '#type': textfield
+        '#title': 'What company were you driving for?'
+        '#required': true
+      report_customers_passengers:
+        '#type': radios
+        '#title': 'Did you have passengers/customers at the time of the crash?'
+        '#options': yes_no
+        '#required': true
+      contact_tnc_permit_number:
+        '#type': textfield
+        '#title': 'TNC permit number'
+        '#required': true
+    report_vehicle_license_plate:
+      '#type': textfield
+      '#title': 'Vehicle License Plate'
+      '#states':
+        visible:
+          ':input[name="report_type"]':
+            checked: true
+      '#required': true
+    report_state_plate_issued:
+      '#type': textfield
+      '#title': 'State Plate Issued'
+      '#states':
+        visible:
+          ':input[name="report_type"]':
+            checked: true
+      '#required': true
+  page_crash_details:
+    '#type': webform_wizard_page
+    '#title': 'Crash Details'
+    portland_location_picker:
+      '#type': portland_location_picker
+      '#title': 'Location of incident'
+      '#place_name__access': false
+      '#location_details__access': false
+      '#required': true
+    report_date_and_time_of_incident:
+      '#type': datetime
+      '#title': 'Date and time of incident'
+      '#required': true
+    report_complaint_detail:
+      '#type': textarea
+      '#title': 'Crash details'
+      '#description': '<p>Please provide a description of the incident.</p>'
+      '#description_display': before
+      '#required': true
+    report_filed_with_state:
+      '#type': radios
+      '#title': 'Was a Report filed with State of Oregon?'
+      '#options':
```

Comment:
This could use the preset `'#options': yes_no` as used elsewhere in the form

Reply:
Fixed